doc/vim-go.txt

```diff
@@ -14,15 +14,16 @@
 ===============================================================================
 CONTENTS                                                          *go-contents*
 
-  1. Intro........................................|go-intro|
-  2. Install......................................|go-install|
-  3. Commands.....................................|go-commands|
-  4. Mappings.....................................|go-mappings|
-  5. Text Objects.................................|go-text-objects|
-  6. Functions....................................|go-functions|
-  7. Settings.....................................|go-settings|
-  8. Troubleshooting..............................|go-troubleshooting|
-  9. Credits......................................|go-credits|
+  1.  Intro........................................|go-intro|
+  2.  Install......................................|go-install|
+  3.  Commands.....................................|go-commands|
+  4.  GoDef Stack..................................|godef-stack|
```

Comment:
We should not have a separate section for this. This should be under `Commands` just below the current `:GoDef` commands so people can follow it.